crates/ssr/src/tests.rs

```diff
@@ -159,6 +159,50 @@ fn assert_match_failure_reason(pattern: &str, code: &str, snippet: &str, expecte
     assert_eq!(reasons, vec![expected_reason]);
 }
 
+#[test]
+fn ssr_let_stmt_in_macro_match() {
+    assert_matches(
+        "let a = 0",
+        r#"
+            macro_rules! m1 { ($a:stmt) => {$a}; }
+            fn f() {m1!{ let a = 0 };}"#,
+        // FIXME: Whitespace is not part of the matched block
+        &["leta=0"],
```

Comment:
Hmm, that is indeed unfortunate. Looks like it's a problem with Match::matched_text. It calls `.text()` on the matched node, which doesn't work inside of macro expansions. Fortunately Match::matched_text is mostly only used from tests, although the command line does use it, but that's mostly for debugging purposes.

Fixing it probably involves passing in the original source and uses ranges to extract the matched text.

Reply:
Doesn't sound like that should be addressed soon :) - and wasn't intending to look into it in this PR anyway. Leaving the `FIXME` in place should be fine, right?